projects/1.20/assets/the-undergarden/undergarden/lang/zh_cn.json

```
@@ -0,0 +1,531 @@
+{
+  "advancement.undergarden.all_ore_blocks.desc": "获取深园中的每种矿石",
+  "advancement.undergarden.all_ore_blocks.title": "典藏系列",
+  "advancement.undergarden.all_undergarden_biomes.desc": "发现所有的深园生物群系",
+  "advancement.undergarden.all_undergarden_biomes.title": "地下制图者",
+  "advancement.undergarden.catacombs.desc": "进入下僻巢穴",
+  "advancement.undergarden.catacombs.title": "遗忆府第",
+  "advancement.undergarden.catalyst.desc": "制作深园催化石",
+  "advancement.undergarden.catalyst.title": "请给张票",

```

```suggestion
  "advancement.undergarden.catalyst.title": "求票记",
```
不确定是否典出同名电影 _One Ticket Please（求票记）_

---

这个感觉也是日常用语

---

主要是谷歌搜索One Ticket Please，第一页基本全是这电影

---

也是
不过日常中也算是常见表达，比较麻烦

---

求张票（

---

```suggestion
  "advancement.undergarden.catalyst.title": "求张票",
```
也行的，ZY这样OK吗

---

都行，其实也不是多重要的东西（）

---

确实